assets/src/edit-story/app/story/useStoryReducer/reducers/setBackgroundElement.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Internal dependencies
+ */
+import { moveArrayElement } from './utils';
+
+/**
+ * Set background element on the current page to the given id.
+ *
+ * If element id is null, background id is cleared for the page.
+ *
+ * If page had a background element before, that element is deleted!
+ * And if that element was selected, selection is cleared.
+ *
+ * @param {Object} state Current state
+ * @param {Object} payload Action payload
+ * @param {number} payload.elementId Element id to set as background on the current page.
+ * @return {Object} New state
+ */
+function setBackgroundElement( state, { elementId } ) {
```

Comment:
The typical pattern so far was to pass `pageId` that could be `null` to default to `state.current`. Should we do this here as well?

Reply:
Could a background element ever be set to a non-current Page? In this case, would it ever be possible to `addElements` to the non-current page, too?

Reply:
I actually asked @samitron7 about editing elements on non-current pages, and she said it wasn't possible in any way. I thus removed all references to pageId on all element editing functions. I can add them, but I don't think we'll ever need them. You'll see the exact same thing for `deleteElements`, `addElements`, `updateElements` and `arrangeElement`.

Reply:
Makes sense to me!